privacyidea/static/components/token/views/token.html

```diff
@@ -1,9 +1,50 @@
-<div class="well" ng-controller="tokenController">
+<div class="well" ng-controller="tokenMenuController">
```

Comment:
somehow the `tokenMenuController` does not get registered when logged in as a user.
I get a bunch of:
```
Error: [$controller:ctrlreg] The controller with the name 'tokenMenuController' is not registered.
```
in the browser console when logging in as a user.